### PR TITLE
Allow php 8

### PR DESCRIPTION
**Pull request by bronek89**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "php": "^7.4"
+        "php": ">=7.4"
```
Comment (dlakomski) [inline composer.json]: I would rather set it to `^7.4 || 8.0.*`

Comment (bronek89) [reply composer.json]: It looks like version with `||` is more widespread:

https://github.com/symfony/symfony/blob/5.x/composer.json#L19

https://github.com/symfony/symfony/blob/5.x/composer.json#L19
https://github.com/thephpleague/oauth2-client/blob/master/composer.json#L9
https://github.com/laravel/laravel/blob/8.x/composer.json#L11

But maybe `^7.4 || ^8.0` would be better?

Comment (dlakomski) [reply composer.json]: Hmm... there shouldn't be any breaking changes in 8.1, so I'm ok with that.

Comment (bronek89) [reply composer.json]: ok, changed ;)

**Comment by dlakomski** [comment]
We need to add php 8 to travis config. Althought it will not work at this moment.

**Comment by bronek89** [comment]
> We need to add php 8 to travis config. Althought it will not work at this moment.

I've just added it to travis, I think it's this PR https://github.com/php-build/php-build/pull/651 so it should be ready soon

**Review by dlakomski — APPROVED** [review]

**Pull request merged**